test/mlir/onnx/parse/test_abs.json.mlir

```diff
@@ -0,0 +1,78 @@
+// RUN: sed 's/^[ \t]*\/\/.*//' %s | onnx-mlir --EmitONNXBasic --printIR | FileCheck %s
```

Comment:
I couldn't get `cmake --build . --target check-onnx-lit` to run the lit tests without adding the extra `.mlir` extension to the lit tests

(but an added benefit of the `.mlir` extension is that vscode syntax highlighting is nicer, vscode marks up the `//` comments in red because they are invalid in json)

Reply:
I figured out how to add '.json' to test/mlir/lit.cfg.py to run .json lit tests and I renamed all the parse tests to .json extension